tests/app/models/BaseModelTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+use DB\SQL;
+
```

Comment:
Codeacy is wanting a namespace for these new classes.  It seems like the convention for PHP and PHPUnit is to use the same namespace as the class being tested.  How do you feel about this?

http://stackoverflow.com/questions/12117254/organizing-phpunit-tests-in-namespaces